plugins/services/src/js/utils/FrameworkUtil.js

```diff
@@ -47,17 +47,20 @@ const FrameworkUtil = {
    * @return {Object}               containing urls from the extracted query string
    * if all is not available in given object.
    */
-  extractImageUrls(packageImages) {
-    const images = Object.assign({}, packageImages);
+  extractImageUrls(packageImages = {}) {
```

Comment:
Should this method be moved to a place that highlights the fact it should only be used in development? May be worth noting that MarathonStore is the only caller.

Or perhaps it's better to tolerate the transformation in production.